doc/user_guide.md

```diff
@@ -11,4 +11,7 @@ That will bring you to:
 
 If Numberscope experiences an error, you might see an alert with the error
 message. If you see one that does not resolve upon reloading the page and
-trying again, please email the address shown with the error message.
+trying again, please let us know about the problem by [filing an issue
+on GitHub](https://github.com/numberscope/frontscope/ issues/new). Please
```

Comment:
space makes this link not work

Reply:
Darn linter. I think I've now got this to a state it is satisfied with.